### PR TITLE
fix: change background for username+password login link

### PR DESCRIPTION
**Pull request by nidalaa**

### CODE REVIEW

**Comment by nidalaa** [comment]
@thomasthesecond  is in comment here: https://github.com/lonelyplanet/backpack-ui/pull/722#issuecomment-452701911

File: src/components/multiStep/multiStepLogin.jsx [hunk 1]
```diff
@@ -28,6 +28,7 @@ const styles = {
     fontWeight: fontWeightMedium,
   },
   legacyButton: {
+    "background-color": "white",
```
Comment (thomasthesecond) [inline src/components/multiStep/multiStepLogin.jsx]: Can we do `backgroundColor: colors.bgPrimary` here?

Comment (nidalaa) [reply src/components/multiStep/multiStepLogin.jsx]: 👍

**Comment by nidalaa** [comment]
Fixed, thanks! :)

**Pull request merged**